zeitschrift-fur-religionswissenschaft-note.csl

```diff
@@ -0,0 +1,1235 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="never" page-range-format="chicago">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Zeitschrift für Religionswissenschaft (notes)</title>
```

Comment:
@frederik-elwert, can you change this to `<title>Zeitschrift für Religionswissenschaft (note)</title>`?